src/features/journeys/components/JourneyInstancesDataTable/getTagColumns.tsx

```diff
@@ -44,38 +41,29 @@ const doesNotHave = (
   col: JourneyTagGroupColumn | JourneyUnsortedTagsColumn,
   item: GridFilterItem
 ) => {
-  return (
-    params: GridCellParams<ZetkinJourneyInstance['tags'], ZetkinJourneyInstance>
-  ) => {
+  return (params: GridCellParams<ZetkinJourneyInstance>) => {
     if (!item.value) {
       return true;
     }
-
     const tags = col.tagsGetter(params.row.tags);
 
-    return !!tags.find((tag) => {
-      return tag.id.toString() !== item.value;
+    return !tags.some((tag) => {
+      return tag.id.toString() == item.value;
```

Comment:
At first I thought the logical change here was a mistake, but I understood it after a while. This looks good, but why change from doing a !== to a == rather than a ===? Not a big thing, but the === doesn't implicitly typecast.

Reply:
I think I just removed the `!` without thinking further, `===` is definitely what it should be!